### PR TITLE
dgs: add u2i data generator and related tutorial doc

### PR DESCRIPTION
**Pull request by goldenleaves**

### CODE REVIEW

**Review by Seventeen17 — APPROVED** [review]
LGTM

File: docs/en/dgs/deploy.md [hunk 1]
```diff
@@ -179,8 +179,8 @@ All workers have the same parameters of common pod assignment, worker-type={coor
 | `sampling.samplePublishing.producerPoolSize`       | The max number of kafka producer for sampling results                                                | `2`                              |
 | `sampling.samplePublishing.maxProduceRetryTimes`   | The maximum retry times of producing a kafka message                                                 | `3`                              |
 | `sampling.samplePublishing.callbackPollIntervalMs` | The interval(ms) for polling async producing callbacks                                               | `100`                            |
-| `sampling.logging.dataLogPeriod`                   | Specify how many graph update batches should be processed between two logs produced                  | `1`                              |
-| `sampling.logging.ruleLogPeriod`                   | Specify how many sampling rules should be processed between two logs produced                        | `1`                              |
+| `sampling.logging.dataLogPeriod`                   | Specify how many graph update batches should be processed between two logs produced                  | `10`                              |
```
Comment (LiSu) [inline docs/en/dgs/deploy.md:182]: between two logs produced --> between two logs

File: docs/en/dgs/deploy.md [hunk 2]
```diff
@@ -179,8 +179,8 @@ All workers have the same parameters of common pod assignment, worker-type={coor
...
+| `sampling.logging.ruleLogPeriod`                   | Specify how many sampling rules should be processed between two logs produced                        | `10`                              |
```
Comment (LiSu) [inline docs/en/dgs/deploy.md:183]: between two logs produced --> between two logs

File: docs/en/dgs/deploy.md [hunk 3]
```diff
@@ -211,7 +211,7 @@ All workers have the same parameters of common pod assignment, worker-type={coor
 | `serving.recordPolling.threadNum`                 | The thread number for sample update consuming from kafka queues                                     | `2`                              |
 | `serving.recordPolling.retryIntervalMs`           | The retry interval (ms) when no record has been polled                                              | `100`                            |
 | `serving.recordPolling.processConcurrency`        | The max processing concurrency for polled records                                                   | `100`                            |
-| `serving.logging.dataLogPeriod`                   | Specify how many sample update batches should be processed between two logs produced                | `1`                              |
+| `serving.logging.dataLogPeriod`                   | Specify how many sample update batches should be processed between two logs produced                | `10`                              |
```
Comment (LiSu) [inline docs/en/dgs/deploy.md:214]: between two logs produced --> between two logs

File: docs/en/dgs/tutorial.md [hunk 4]
```diff
@@ -1,38 +1,56 @@
...
+
 ```shell
-cd examples/data
-python cora.py
+cd dynamic_graph_service
+python3 python/data/u2i/u2i_generator.py --output-dir /tmp/u2i_gen
 ```
 
-TODO(@Seventeen17): use open streaming data source
+This generator with first create a static graph for bulk-loading data, which will be used for offline training.
```
Comment (LiSu) [inline docs/en/dgs/tutorial.md:28]: with first --> will first

File: docs/en/dgs/tutorial.md [hunk 5]
```diff
@@ -1,38 +1,56 @@
...
+After that, the streaming data will consume the bulk-loading data with timestamp 0 as an initial state, and further generate graph updates on an increased timestamp.
```
Comment (LiSu) [inline docs/en/dgs/tutorial.md:29]: While generating the streaming data, the previously generated bulk-loading data will be used as the initial state of the graph whose timestamp set to 0.  After that, we generate more graph updates with increasing timestamps.

File: docs/en/dgs/tutorial.md [hunk 6]
```diff
@@ -1,38 +1,56 @@
...
+The generated data is in `/tmp/u2i_gen/training` and `/tmp/u2i_gen/streaming` respectively.
```
Comment (LiSu) [inline docs/en/dgs/tutorial.md:30]: is in --> are stored in

File: docs/en/dgs/tutorial.md (hunk 5 above)
Comment (goldenleaves) [reply docs/en/dgs/tutorial.md]: done

File: docs/en/dgs/deploy.md (hunk 1 above)
Comment (goldenleaves) [reply docs/en/dgs/deploy.md]: fixed

File: docs/en/dgs/deploy.md (hunk 2 above)
Comment (goldenleaves) [reply docs/en/dgs/deploy.md]: fixed

**Review by LiSu — APPROVED** [review]

File: docs/en/dgs/deploy.md (hunk 3 above)
Comment (goldenleaves) [reply docs/en/dgs/deploy.md]: fixed

File: docs/en/dgs/tutorial.md (hunk 4 above)
Comment (goldenleaves) [reply docs/en/dgs/tutorial.md]: fixed

**Pull request merged**